### PR TITLE
Waverunner: Add functions for load/store of setups to local file

### PR DESCRIPTION
**Pull request by johannheyszl**
incl. test file

### CODE REVIEW

File: cw/waverunner.py [hunk 1]
```diff
@@ -92,7 +94,71 @@ def _ask(self, cmd):
...
+            "Unrecognized keyword.", 
+            "String error.", 
+            "GET embedded in another message.", 
+            "Arbitrary data block expected.", 
+            "Non-digit character in byte count field of arbitrary data block.", 
+            "EOI detected during definite length data block transfer.", 
+            "Extra bytes detected during definite length data block transfer"]
+        return_code = int((re.findall(r'\d+', self._ask("CMR?")))[0])
+        if return_code > 11:
+            return_msg = f"{return_code} Unkown error code"
```
Comment (vrozic) [inline cw/waverunner.py:115]: Looking at the table in the manual, the codes are numbered 1..7 and 10..13, while codes 8 and 9 are unused.

**Comment by vrozic** [comment]
Thanks @johannheyszl 
This looks mostly good FWICT, but of course I cannot test it with an oscilloscope.
I only noticed that the error codes may be inconsistent with documentation (comment above)

CI failures are mostly due to `flake8` complaining about indentation, trailing white spaces and imported-but-unused modules.

File: cw/waverunner.py (hunk 1 above)
Comment (johannheyszl) [reply cw/waverunner.py:115]: mega - thanks for catching this!

File: cw/waverunner.py [hunk 2]
```diff
@@ -93,6 +95,74 @@ def _ask(self, cmd):
...
+                     "String error.",
+                     "GET embedded in another message.",
+                     "",
+                     "",
+                     "Arbitrary data block expected.",
+                     "Non-digit character in byte count field of arbitrary data block.",
+                     "EOI detected during definite length data block transfer.",
+                     "Extra bytes detected during definite length data block transfer"]
+        return_code = int((re.findall(r'\d+', self._ask("CMR?")))[0])
+        if return_code > 11:
```
Comment (vrozic) [inline cw/waverunner.py:118]:
```suggestion
        if return_code > 13 or return_code in [8, 9]:
```

File: cw/test_waverunner.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Test WaveRunner class."""
+
+from datetime import datetime
+from waverunner import WaveRunner
```
Comment (vrozic) [inline cw/test_waverunner.py:9]:
```suggestion
from datetime import datetime

from waverunner import WaveRunner
```

I'm afraid that the remaining lint error message is not very informative.
You can run `isort -c -w79 test_waverunner.py --diff` locally to see what isort is expecting.
At the moment it is only complaining about the missing empty line.

**Comment by johannheyszl** [comment]
Thanks @vrozic for your help

**Review by vrozic — APPROVED** [review]

**Pull request merged**